config.json

```diff
@@ -753,7 +753,7 @@
       },
       {
         "slug": "pythagorean-triplets",
-        "name": "pythagorean-triplets",
```

Comment:
You also need to undo the filepath. The directories should match the slug.

Reply:
Forgot about it)